docs/TESTS.md

```diff
@@ -28,38 +28,40 @@ Otherwise, the `pytest` installation will be global.
 
 ```powershell
 PS C:\Users\foobar> py -m pip install pytest pytest-cache pytest-subtests pytest-pylint
-Successfully installed pytest-7.2.2 ...
+Successfully installed pytest-8.3.3 ...
 ```
 
 #### Linux / MacOS
 
 ```bash
 $ python3 -m pip install pytest pytest-cache pytest-subtests pytest-pylint
-Successfully installed pytest-7.2.2 ...
+Successfully installed pytest-8.3.3 ...
 
 ```
 
 To check if installation was successful:
 
 ```bash
 $ python3 -m pytest --version
-pytest 7.2.2
+pytest 8.3.3
 ```
 
 ## Running the tests
 
-To run the tests, go to the folder where the exercise is stored using `cd` in your terminal (_replace `{exercise-folder-location}` below with your path_).
+To run the tests, go to the folder where the exercise is stored using `cd` in your terminal (_replace `<exercise-folder-location>` below with your path_).
 
 ```bash
-$ cd {exercise-folder-location}
+$ cd <exercise-folder-location>
 ```
+**Note:** `<exercise-folder-location>` or most things inside angle brackets denote a placeholder value.
+A normal path should simply has its name without any brackets like: `myFolder` or `myFile.py`
```

Comment:
```suggestion

<br>

~~~~exercism/note
 `<exercise-folder-location>` or most things inside angle brackets denote a **_placeholder value_**.
A normal path or file name should be written _without_ any brackets.  


For example: `/Users/janedoe/exercism/python/exercises/concept/chaitanas-colossal-coaster` (on *nix systems), `C:\Users\janedoe\exercism\python\exercises\practice\hello-world\` (on Windows), `myFolder` or `my_file.py`.
~~~~  

<br>

```

Reply:
Nice. I thought about using the exercism note block as well, but I saw up above of the doc that there was a simple **Note** so I thought maybe I should do the simple one as well